src/libraries/System.Text.Encodings.Web/ref/System.Text.Encodings.Web.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
```

Comment:
I believe @joperezr went through all OOB libraries and if they build `netstandard2.0`, he ensured they built `net461`. That way when someone uses this library on .NET Framework, they don't pull in the netstandard facades.